nextflow.config

```diff
@@ -13,6 +13,8 @@ params {
   spectra = "data/*.mzML"
   database = "data/*.fasta"
   //expdesign = "data/*.tsv"
+  adddecoys = false
+  se = "comet"
```

Comment:
maybe write `se` out, since not everyone might easily understand that it's search engine